src/backend/adapters/angular2.ts

```diff
@@ -78,6 +79,18 @@ export class Angular2Adapter extends BaseAdapter {
     ngZone.onStable.subscribe(() => this._onEventDone.next());
   }
 
+  showAppRoutes(): void {
+    const root = this._findRoot();
+    try {
+      const routes = ParseRouter.parseRoutes(
+        ng.probe(root).componentInstance.router.root.registry);
+
+      this.showRoutes(routes);
+    } catch (error) {
+      console.log(error);
```

Comment:
Any parsing errors or inconsistent data due to API changes. Don't want users to get blocked.